### DISCUSSION TITLE
Example request - a custom theme for beta 40+

### DISCUSSION THREAD

**Discussion by jlooper** [vuepress/core#881] [Q&A] [answered]
Hello! I'm working on a site for a book I'm writing: https://github.com/CS4Kids/teachers-site. I'd love to theme it for kids and teachers with some special effects like particles, etc. For this, I need themes to work but after the latest changes my themes are broken, as is expected. Does someone have a dead simple demo of a custom local theme that you can share? I promise to write a blogpost if I can get my themes working again :)

**Comment by meteorlxy** [comment, marked as answer]
Hi, glad to hear that you are writing with vuepress-next!

After the latest breaking changes, we've also updated the docs about extending the default theme, which might be helpful:

https://v2.vuepress.vuejs.org/reference/default-theme/extending.html

Let me know if there are anything unclear, and we'll try to improve the guide :)

**Comment by jlooper** [comment]
hi! wow, that really helped, thanks @meteorlxy - Here's the site, getting back to my styles now https://www.cs4kids.club/ - the only thing is, I've lost my custom font. It's loaded in the palette file, `@import url('https://fonts.googleapis.com/css2?family=Dokdo&family=Fredoka:wght@400&display=swap');` but doesn't seem to appear on the page now. Any ideas?

**Reply by jlooper** [reply]
oh! just figured it out. Google fonts need to go in Config! Yay! we're back in business :)
